src/hashTable.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+import LinkedList from 'linkedlist'
+
+export default class HashTable {
+  //check
+  constructor(){
+    this.tablesize = 5831
```

Comment:
ohh this is your hashing! you can also use a npm package like this to do it: https://www.npmjs.com/package/hash.js